h2o-algos/src/main/java/water/tools/MojoConvertTool.java

```diff
@@ -33,25 +33,28 @@ void convert() throws IOException {
         Files.write(pojoPath, pojo.getBytes(StandardCharsets.UTF_8));
     }
 
-    private static void usage() {
-        System.err.println("java -cp h2o.jar " + MojoConvertTool.class.getName() + " source_mojo.zip target_pojo.java");
+    public static void main(String[] args) throws IOException {
+        try {
+            mainInternal(args);
+        }
+        catch (IllegalArgumentException e) {
+            System.err.println(e.getMessage());
```

Comment:
Please make sure this only print out the actual error message and not the whole stack.